v3/cypress/support/elements/TableTile.ts

```diff
@@ -0,0 +1,126 @@
+export const TableTileElements = {
+    getTableTile() {
+        return cy.get(".codap-component").eq(2);
+    },
+    getCaseTableGrid() {
+        return cy.get("[data-testid=case-table] [role=grid]");
+    },
+    getNumOfAttributes() {
+        return cy.get("[data-testid=case-table] [role=grid]").invoke("attr", "aria-colcount");
+    },
+    getNumOfCases() {
+        return cy.get("[data-testid=case-table] [role=grid]").invoke("attr", "aria-rowcount");
+    },
+    getCollectionTitle() {
+        return this.getTableTile().find("[data-testid=editable-component-title]");
+    },
+    getColumnHeaders() {
+        return cy.get("[data-testid=case-table] [role=columnheader]");
+    },
+    getColumnHeader(index) {
+        return cy.get(".codap-column-header-content").eq(index);
+    },
+    getColumnHeaderTooltip() {
+        return cy.get("[data-testid=case-table-attribute-tooltip]");
+    },
+    openIndexMenuForRow(rowNum) {
+        cy.get("[data-testid=case-table] [role=grid] [role=row][aria-rowindex=\"" + rowNum + "\"] [data-testid=codap-index-content-button]").click();
```

Comment:
```suggestion
        cy.get(`[data-testid=case-table] [role=grid] [role=row][aria-rowindex="${rowNum}"] [data-testid=codap-index-content-button]`).click();
```
Use template strings rather than string concatenation.

Reply:
Done